tests/Common/SearchComponentsConfigurationsTest.php

```diff
@@ -22,14 +26,21 @@ class SearchComponentsConfigurationsTest extends StorageApiTestCase
         ]
     ];
 
-    use ComponentsConfigurationUtils;
+    /**
+     * @var BranchAwareClient
+     */
+    private $client;
 
     public function setUp()
     {
         parent::setUp();
 
         $this->cleanupConfigurations();
-        $this->initEvents();
+
+        $clientProvider = new ClientProvider($this);
+        $this->client = $clientProvider->createBranchAwareClientForCurrentTest();
```

Comment:
Nejsem si jistej, ale nemělo by tady dycky bejt normální get client? Takhle i pokud bude "defaultBrnach" tak to dostane branchovaného klienta, ne?

Reply:
Ale vlastně tam je to jen skrz branch, tak nic, tak je to OK ;)